frameworks/AutoGluon/__init__.py

```diff
@@ -53,8 +65,23 @@ def run_autogluon_timeseries(dataset: Dataset, config: TaskConfig):
         problem_type=dataset.type.name,  # AutoGluon problem_type is using same names as amlb.data.DatasetType
         timestamp_column=dataset.timestamp_column,
         id_column=dataset.id_column,
-        forecast_range_in_steps=dataset.forecast_range_in_steps
+        forecast_horizon_in_steps=dataset.forecast_horizon_in_steps,
+        seasonality=dataset.seasonality
     )
 
     return run_in_venv(__file__, "exec_ts.py",
                        input_data=data, dataset=dataset, config=config)
+
+def read_setup_env_vars():
+    env_vars = {}
+    fpath = os.path.join(os.path.dirname(os.path.realpath(__file__)), '.setup', 'setup_env')
+    try:
+        with open(fpath, 'r') as f:
+            for line in f:
+                if line.startswith('#') or not line.strip():
+                    continue
+                key, value = line.strip().split('=', 1)
+                env_vars[key] = value
+    except OSError:
+        print(f'Could not open/read file {fpath}')
+    return env_vars
```

Comment:
I'd like to avoid starting to see multiply implementations of this.
Please move to `benchmark.py` as `_read_setup_env()` (after `_write_setup_env`)
and then add to `TaskConfig`:
```python
class BenchmarkTask:

    def __init__(self, benchmark: Benchmark, task_def, fold):
        ....
        self.task_config = TaskConfig(
            ...
            output_dir=benchmark.output_dirs.session,
            setup_env=benchmark._read_setup_env(),
        )
```
then you can access those setup vars directly in `__init__` or even in `exec.py`